class RetryError(Exception):
    """Exception raised when a task should be retried."""
    ...
